torch/_dynamo/variables/builder.py

```diff
@@ -135,8 +136,16 @@ class GraphArg:
     # or not.
     is_tensor: bool = True
 
+    @property
+    def example(self):
+        if isinstance(self._example, TensorWeakRef):
+            return self._example()
+        else:
+            return self._example
```

Comment:
assert SymInt so that real tensors don't creep back in?

Reply:
I'd probably prefer to unwind the example/fake_example Tensor/SymInt/FakeTensor confusion